### PR TITLE
feat: added PATH_TO_DB environment variable

### PR DESCRIPTION
**Pull request by DmitriyLewen**
Issue #167

### CODE REVIEW

File: dbservice/dbparam_test.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	"path/filepath"
+	"testing"
+)
+
+func TestSetNewDbPathFromEnv(t *testing.T) {
+	var tests = []struct {
+		env            string
+		expectedDBPath string
+	}{
+		{"", "/server/database/webhooks.db"},
```
Comment (AndreyLevchenko) [inline dbservice/dbparam_test.go:14]: add name describing test case

File: dbservice/dbparam_test.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	defer os.Setenv("PATH_TO_DB", envOld)
+
+	for _, test := range tests {
+		os.Setenv("PATH_TO_DB", test.env)
+		SetNewDbPathFromEnv()
+
+		if test.expectedDBPath != DbPath {
+			t.Errorf("Paths is not equals, expected: %s, got: %s", test.expectedDBPath, DbPath)
+		}
+		os.RemoveAll(filepath.Dir(test.env))
```
Comment (AndreyLevchenko) [inline dbservice/dbparam_test.go:28]: should be in `defer()`. Use `t.Run(...)`

**Review by AndreyLevchenko — CHANGES_REQUESTED** [review]

File: dbservice/dbparam_test.go (hunk 2 above)
Comment (DmitriyLewen) [reply dbservice/dbparam_test.go]: done

File: dbservice/dbparam_test.go (hunk 1 above)
Comment (DmitriyLewen) [reply dbservice/dbparam_test.go]: done

File: dbservice/dbparam.go [hunk 3]
```diff
@@ -25,3 +28,19 @@ func ChangeDbPath(newPath string) {
 	DbPath = newPath
 	mutex.Unlock()
 }
+
+func SetNewDbPathFromEnv() {
+	newPath := os.Getenv("PATH_TO_DB")
+	if newPath != "" {
+		if _, err := os.Stat(newPath); err != nil {
+			if os.IsNotExist(err) {
```
[the remark this answers is not in the document]
Comment (DmitriyLewen) [reply dbservice/dbparam.go:36]: added errors and test for them

**Review by AndreyLevchenko — APPROVED** [review]

**Pull request merged**